### DISCUSSION TITLE
Semver Commit Messages

### DISCUSSION THREAD

**Discussion by cliedelt** [renovatebot/renovate#12723] [Request Help] [answered]
I'm looking for a way to not use "semantic commits" messages but messages following the semantic versioning pattern.

For example my current merge request title (commit message) is: chore(deps): update dependency @types/jasmine to v3.10.2
It updates the version ^3.6.9 -> 3.10.2.

But I want it to use semantic versioning (MAJOR.MINOR.PATCH). So the correct commit message would be: "patch(deps): update dependency ..."

**Comment by rarkins** [comment, marked as answer]
You can create some packageRules which use both `matchUpdateTypes` and `semanticCommitType` to define the mapping.
